### ISSUE TITLE
Error when retrieving data of focused row

### ISSUE THREAD

**Issue by yangyeongwon** [ohah/vue3-tui-grid#5]
Hi,

An error occurs when calling the 'getRow' function to get the data of the focused row

#version
vue 3.2.27
tui-grid 4.21.2

# my code

```
const GridTable = ref();
let grid = ref();     
let instance = ref();

onMounted(() => {
 grid = GridTable.value;
 instance = grid.gridInstance;
});

const handleEdit = () => {
 const {rowKey} = instance.getFocusedCell();
 console.log(instance.getRow(0));
}

```

# error
![image](https://user-images.githubusercontent.com/66061229/198523164-b0a70fb4-e14d-4737-b33e-a374869ae372.png)

**Comment by ohah** [comment]
thank you
fix it over the weekend

**Comment by yangyeongwon** [comment]
Hi, 
Is the problem resolved?

**Comment by ohah** [comment]
@yangyeongwon fixed v0.151
